examples/ci-tested-notebooks.txt

```diff
@@ -6,7 +6,6 @@ compilation_example
 conditional_gate_example
 contextual_optimization
 creating_backends
-Forest_portability_example
```

Comment:
Should we also move the notebook itself? Or were you planning to do that in a separate PR?

Reply:
You mean to the legacy directory? I would have hoped that we would be able to add it back in soon and can just keep it here?